OriginateTheme/Swift/Models/Fonts.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Fonts.swift
+//  OriginateTheme
+//
+//  Created by Robert Weindl on 2017-02-14.
+//  Copyright © 2017 Originate Inc. All rights reserved.
+//
+
+import UIKit
+
+struct Fonts {
+    var dictionary: ThemeDefinition
+
+    public var body: UIFont {
```

Comment:
I think this an example of what the generated properties would look like?  He mentioned he added examples.